inc/vcf/record_cache.hpp

```diff
@@ -114,7 +158,10 @@ namespace ebi
         }
 
       private:
-        std::multiset<RecordCore> cache;
+        std::multiset<RecordCore> cache_duplicates;
+        std::multiset<RecordCore> cache_symbolic_duplicates;
+        std::vector<Error*> list_duplicates{};
+        std::vector<Error*> list_symbolic_duplicates{};
```

Comment:
not sure what is the reason for the segfault yet, but if you revert this back to std::vector<std::unique_ptr<Error>> it works